src/test/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/helper/EntityDataLoader.kt

```diff
@@ -409,11 +414,12 @@ class VisitEntityHelper(
 @Transactional
 class EventAuditEntityHelper(
   private val eventAuditRepository: TestEventAuditRepository,
+  private val testActionedByRepository: TestActionedByRepository,
```

Comment:
Why do we have Test repositories? Why don't we use the actual repositories like we do in other Integration tests?